### DISCUSSION TITLE
Setting font size

### DISCUSSION THREAD

**Discussion by jernejs** [uowuo/abaddon#232] [Q&A]
Is there any way to set the font size globally? The text in Abaddon is just a bit too small for me.

**Comment by ouwou** [comment]
which OS are you on? if linux, i hesitate to give an answer cuz im not sure what the "right" one would be but id google something like "change gtk application font size linux"
if windows, in theory you should be able to create a `settings.ini` file in the path `etc/gtk-3.0/settings.ini` (etc already exists in the release build) with contents:

```ini
[Settings]
gtk-xft-dpi=98304
```

gtk-xft-dpi is DPI * 1024, so 98304 is standard 96 * 1024. 122880 is 1.25x that, etc.

ill probably add a more convenient setting so you can just put in a scaling factor.
alternatively you can probably add a css rule like `* { font-size: 1.5em; }` but i didnt test that

**Comment by jernejs** [comment]
I forgot to mention, I'm on Windows. `settings.ini` didn't seem to have any effect (tried it with values up to 307200), but `* { font-size: 101%; }` did the trick.

**Comment by ouwou** [comment]
i added a setting that does what `etc/gtk-3.0/settings.ini` does, but 1 is unscaled, 2 is 2x etc. if CSS works fine for u thats fine ive been meaning to add a real setting for it for a while now anyways :p

**Reply by eeriemyxi** [reply]
Did you add any setting? Can you help me set my font size please? I don't understand the thread here at all.
